src/adapter/src/session/vars.rs

```diff
@@ -379,6 +379,15 @@ const PERSIST_COMPACTION_MINIMUM_TIMEOUT: ServerVar<Duration> = ServerVar {
     safe: true,
 };
 
+/// Controls [`mz_persist_client::cfg::DynamicConfig::consensus_connect_timeout`].
+const PERSIST_CONSENSUS_CONNECT_TIMEOUT: ServerVar<Duration> = ServerVar {
```

Comment:
Does this need to be changeable at runtime? If not, it should be a param on envd's config, not a SystemVar.

Reply:
Are there considerations to make before adding new SystemVars? Updating the value wouldn't affect connections until the process restarts, but I would prefer being able to set it dynamically without a rebuild in case we get the value wrong or need to increase it in case of some odd network incident.

Reply:
Ah, that sounds fine for a system var then. I *think* that making it a SystemVar like this means you can set it via LaunchDarkly? (I'm not quite sure how that works.)

One other request then: can you rename this to be more general so that the stash can also use it? I think it should probably always be the same value.

Reply:
👍 sure thing, will update